python/lsst/pipe/tasks/calibrateImage.py

```diff
@@ -410,11 +410,6 @@ def setDefaults(self):
         self.photometry.match.sourceSelection.doRequirePrimary = False
         self.photometry.match.sourceSelection.doUnresolved = False
 
-        # All sources should be good for PSF summary statistics.
-        # TODO: These should both be changed to calib_psf_used with DM-41640.
-        self.compute_summary_stats.starSelection = "calib_photometry_used"
-        self.compute_summary_stats.starSelector.flags.good = ["calib_photometry_used"]
```

Comment:
Why is this removed and not changed to (e.g.) `calib_psf_used`?

Reply:
`calib_psf_used` is already the default.